src/main/webapp/create-live-stream.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+	<head>
```

Comment:
This might be a GitHub issue, but the indentation is off for me here.

Reply:
Fixed